src/content/blog/flutter-jobs.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Getting a Flutter Job
+author: eseidel
+description: How to get a Flutter Job
+date: 2024-10-10
+---
+
+I founded the Flutter project 10 years ago, and lead the Flutter team at Google
+for 8 years. I'm reasonably well known in the Flutter community and receive a
+lot of email from both job-seekers and job-posters regarding Flutter.
+
+I've also been a manager for a decade and can tell you, hiring is hard. From
+both sides. It's hard for companies to find candidates, and it's hard for
+candidates to find companies.
+
+This can be made extra-hard when trying to search for a subset of jobs which
+include a specific technology choice, like Flutter.
+
+I generally try to dissuade both job-seekers and job-posters to search
+specifically for Flutter, under the belief that Flutter is a tool -- a means to
+an end -- and like other tools will come and go in popularity over time and what
+we all (ideally) focus on in our jobs is solving customer problems. As
+technologists of course, we learn whatever tools we need to do accomplish that
+goal including Flutter.
+
+The difference is subtle, but I believe important. Most places that should be
+using Flutter, may not know they should be using Flutter (because they haven't
+hired the person to make that choice yet). Similarly many developers who will be
```

Comment:
```suggestion
The difference is subtle, but I believe important. Most places that should be
using Flutter may not know they should be using Flutter (because they haven't
hired the person to make that choice yet). Similarly, many developers who will be
```